webapi/Services/AzureContentSafety.cs

```diff
@@ -136,17 +136,13 @@ public async Task<ImageAnalysisResponse> ImageAnalysisAsync(IFormFile formFile,
         var body = await response.Content.ReadAsStringAsync(cancellationToken).ConfigureAwait(false);
         if (!response.IsSuccessStatusCode || body is null)
         {
-            throw new AIException(
-                response.StatusCode == System.Net.HttpStatusCode.Unauthorized ? AIException.ErrorCodes.AccessDenied : AIException.ErrorCodes.UnknownError,
-                $"[Content Safety] Failed to analyze image. {response.StatusCode}");
+            throw new SKException($"[Content Safety] Failed to analyze image. {response.StatusCode}");
         }
 
         var result = JsonSerializer.Deserialize<ImageAnalysisResponse>(body!);
         if (result is null)
         {
-            throw new AIException(
-                AIException.ErrorCodes.UnknownError,
-                $"[Content Safety] Failed to analyze image. {body}");
+            throw new SKException($"[Content Safety] Failed to analyze image. {body}");
```

Comment:
I know it was like this before, but do we really want to log the WHOLE image's contents if we hit an error?  That seems like it could be a lot of data.  Maybe just the file name would be sufficient?

Reply:
![image](https://github.com/microsoft/chat-copilot/assets/125500434/4f57cf75-565b-439b-8857-5e87d8cf47b8)
body in this context isn't the image's content. It's the response body, so it should be some form of the ImageAnalysisResponse or an error message.

Reply:
OK!